src/coreclr/tools/superpmi/superpmi/jitinstance.cpp

```diff
@@ -245,7 +200,7 @@ bool JitInstance::reLoad(MethodContext* firstContext)
     FreeLibrary(hLib);
```

Comment:
`JitInstance::reLoad` looks like a dead code and it can be deleted. Once you do that, `PathToOriginalJit` field becomes unused and it can be deleted too.

Reply:
@dotnet/jit-contrib Any comment on the functionality of superpmi?

Reply:
As stated, `reLoad` can be deleted since it is dead code.